lib/profiling/lfric/README.md

```diff
@@ -0,0 +1,40 @@
+# PSyclone Wrapper Library for LFRic
+
+This is a wrapper library that maps PSyclone's PSyData profiling API
+to the timer functionality provided in LFRic.
+
+## Dependencies
+
+While this wrapper library uses LFRic infrastructure code for the timer,
+the library itself can be used independent of LFRic for any code. The
+``Makefile`` will create two different versions of the library:
+- ``libpsy_lfric_timer.a``
+
+  This version needs to be linked with the
+  LFRic infrastructure library, which provides the actual timer
+  code (and dependencies for the timer). This version is meant to be
+  used for LFRic, since the corresponding objects files are already
+  included in the LFRic build.
+
+- ``libpsy_lfric_timer_standalone.a``
+
+  This version of the library contains all dependencies to use the
+  LFRic timer and can be used with any application. It uses the files
+  of the simplified LFRic infrastructure library contained in LFRic.
+
+
+## Compilation
+
+Compilation is done with ``make``. You can adjust the variables
+``F90`` and ``F90FLAGS`` to point to your Fortran compiler and the
+compiler flags you want to use.
+
+
+Example output:
```

Comment:
Probably worth saying that the output is written to `timer.txt`.

Reply:
Good idea.